src/components/profileEdit/AddressInfo.tsx

```
@@ -87,14 +97,46 @@ export default function AddressInfo() {
+                value: detailAddress,
+                error: ""
+            }
+        } else if (detailAddress.length > ADDRESS_DETAIL_MAX_LENGTH) {
+            return {
+                value: detailAddress.slice(0, ADDRESS_DETAIL_MAX_LENGTH),
+                error: result.error?.issues[0].message
+            }
+        } 
+        return {
+            value: "",
+            error: result.error?.issues[0].message || ""
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)